### PR TITLE
STORM-1604:Delayed transition should handle NotALeaderException

### PR DESCRIPTION
**Pull request by Parth-Brahmbhatt**

### CODE REVIEW

**Comment by harshach** [comment]
+1

File: storm-core/src/clj/org/apache/storm/daemon/nimbus.clj [hunk 1]
```diff
@@ -1334,7 +1334,7 @@
      (.unset_reset_log_level_timeout_epoch log-config))))
 
 (defmethod blob-sync :distributed [conf nimbus]
-  (if (not (is-leader nimbus :throw-exception false))
+  (if (not (is-leader nimbus :throw-exception false
```
Comment (abellina) [inline storm-core/src/clj/org/apache/storm/daemon/nimbus.clj]: Looks like a missing parenthesis

**Comment by ptgoetz** [comment]
This doesn't compile, as @abellina pointed out. Adding back the parens on line 1337 fixed the compilation error and all tests pass.

**Comment by harshach** [comment]
@Parth-Brahmbhatt can you fix the above mentioned issue.

**Comment by Parth-Brahmbhatt** [comment]
@harshach will update tonight.

**Comment by Parth-Brahmbhatt** [comment]
@harshach fixed.

**Comment by harshach** [comment]
Thanks @Parth-Brahmbhatt

**Comment by harshach** [comment]
Merged into 1.x-branch

**Pull request merged**